changelog.d/11527.misc

```diff
@@ -0,0 +1 @@
+Temporarily revert usage of `matrix-python-common`.
```

Comment:
Can we nuke both changelogs after this merges so we don't add anything to the changes doc?

Reply:
Yep, I'll be cutting them out of the changes doc after.